File: Herencia/ejercicio 2/Python/Lavadora.py
from Electrodomestico import Electrodomestico
class Lavadora(Electrodomestico):
    def __init__(self, precioBase, color, consumoEnergico, peso, carga):
        super().__init__(precioBase, color, consumoEnergico, peso)
        self.carga = carga
    def precioFinal(self):
        precio = super().precioFinal()
        if self.carga > 30:
            precio += 50
        return precio

File: Herencia/ejercicio 2/Java/Lavadora.java
class Lavadora extends Electrodomestico {
    public double carga;
    public Lavadora(double precioBase, String color, char consumoEnergetico, double peso, double carga) {
        super(precioBase, color, consumoEnergetico, peso);
        this.carga = carga;
    }
    @Override
    public double precioFinal() {
        double precio = super.precioFinal();
        if (carga > 30) precio += 50;
        return precio;
    }
    public double getCarga() {
        return carga;
    }
}